doc/cfgfile.rst

```diff
@@ -40,8 +40,15 @@ Application section
    Ensure that this boilerplate code is at the top of your script::
 
        #!python3.6
-       import sys
-       sys.path.insert(0, 'pkgs')
+       import sys, os
+       import site
+
+       scriptdir, script = os.path.split(__file__)
+       installdir = scriptdir  # for compatibility with commands
```

Comment:
```suggestion
```

This is a bit of a distraction for custom scripts - it's in the standard script template to allow for the custom 'extra preamble' bit, but you don't use that if you're writing a custom script anyway.